packages/commonwealth/client/scripts/views/modals/new_snapshot_proposal_modal.tsx

```diff
@@ -23,16 +23,23 @@ export const NewSnapshotProposalModal = ({
   onModalClose,
 }: NewSnapshotProposalModalProps) => {
   const [selectedSnapshotId, setSelectedSnapshotId] = useState<string | null>(
-    null
+    null,
   );
+  const handleSelect = (item) => {
+    setSelectedSnapshotId(item.value);
+    const init = async () => {
+      await app.snapshot.init(item.value);
+    };
+    init();
```

Comment:
what's the reason for this weird construction? can't you just call `app.snapshot.init(item.value)`?

Reply:
@masvelio Just pushed changes, thanks